automol/util/vector.py

```diff
@@ -69,40 +68,39 @@ def are_parallel(xyz1, xyz2, orig_xyz=(0.0, 0.0, 0.0), anti=False, tol=1e-3):
 
 
 def orthogonalize(xyz1, xyz2, normalize=False):
-    """orthogonalize `xyz2` against `xyz1`"""
-    overlap = numpy.dot(xyz1, xyz2)
-    norm = numpy.dot(xyz1, xyz1)
-    oxyz2 = numpy.subtract(xyz2, numpy.multiply(overlap / norm, xyz1))
+    """Orthogonalize `xyz2` against `xyz1`.
+    :param xyz1: 3D vector
+    :param xyz2: 3D vector.
+    """
+    overlap = np.dot(xyz1, xyz2)
+    norm = np.dot(xyz1, xyz1)
+    oxyz2 = np.subtract(xyz2, np.multiply(overlap / norm, xyz1))
     if normalize:
         oxyz2 = unit_norm(oxyz2)
     return oxyz2
 
 
-def flip_if_left_handed(xvec, yvec, zvec) -> Vector:
+def flip_if_left_handed(xvec: Vector, yvec: Vector, zvec: Vector) -> Vector:
```

Comment:
For the inputs, let's say `Sequence | Vector` as above. (Throughout, we can change `np.ndarray` types to `Vector`).